Dockerfile

```diff
@@ -22,20 +22,24 @@ RUN apk add --update \
     curl-dev \
  && rm -rf /var/cache/apk/* \
  && bundle config --global frozen 1 \
- && bundle install --without test production --jobs 2 \
+ && bundle install --without test --jobs 2 \
```

Comment:
Thinking along similar lines, if we want people to be able to use the base docker image to do all the things that Octobox can do, should that also include run the test suite? For one thing that would mean we could integrate the Docker image into the CI pipeline

Reply:
Maybe? That sounds right. I'll leave adding the test stuff as a separate change though (to be included on any other changes needed to be made by someone getting that all set up, if running the Docker image in the CI pipeline is a route you want to go).